src/android/ChromeCustomTabPlugin.java

```diff
@@ -87,6 +89,33 @@ public boolean execute(String action, JSONArray args, CallbackContext callbackCo
                 callbackContext.sendPluginResult(pluginResult);
                 return true;
             }
+            case "getViewHandlerPackages": {
+                PluginResult pluginResult;
+                final JSONObject json = new JSONObject();
+                json.put("defaultHandler", CustomTabsHelper.getDefaultViewHandlerPackageName(context));
+                json.put("customTabsImplementations", new JSONArray(CustomTabsHelper.getPackagesSupportingCustomTabs(context)));
+                pluginResult = new PluginResult(PluginResult.Status.OK, json);
+                callbackContext.sendPluginResult(pluginResult);
+                return true;
+            }
+            case "useCustomTabsImplementation": {
+                PluginResult pluginResult;
+                JSONObject json = new JSONObject();
+                final String packageName = args.optString(0);
+                if(TextUtils.isEmpty(packageName)) {
+                    json.put("error", "blank packageName");
```

Comment:
_Nit_ : if we're hoping to get this merged back upstream, it may be nice to use the existing [naming](https://github.com/doxo/cordova-plugin-safariviewcontroller/pull/1/files#diff-28d5d0b98f1274f0850ba29552b95e14R73) & [error message formatting](https://github.com/doxo/cordova-plugin-safariviewcontroller/pull/1/files#diff-28d5d0b98f1274f0850ba29552b95e14R59) 

So renaming the `json` variable to `result`, and changing the error message when the `packageName` is empty to be `"expected argument 'packageName' to be non empty string."`